bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/ManagedRuleProvider.java

```diff
@@ -21,8 +28,20 @@
  * @author Ana Dimova - Persistence implementation
  * @author Kai Kreuzer - refactored (managed) provider and registry implementation
  */
+@Component(service = { RuleProvider.class, ManagedRuleProvider.class }, immediate = true)
```

Comment:
From my understanding rule providers are now consumed by the rule registry. So why using immediate = true anymore?